packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -134,6 +154,19 @@ export class GuxAvatar {
     ) as JSX.Element;
   }
 
+  private renderCameraIcon(): JSX.Element {
+    if (this.size === 'large' && this.changePhoto) {
```

Comment:
I am wondering does the size equal large check matter ? I know monse said it was designed specifically for large and the reasoning being is that is only possible to change your profile photo in the profile area of Genesys cloud where the image is large but I dont see why we should limit it to large.

Reply:
Hi Gavin, I've spoken to UX about the large avatar change photo thing again based on your comment. Their point of view is that the change photo icon on a smaller avatar represents bad UX so they'd rather not have it there.